server/tests/fixtures/peertube-plugin-test-four/main.js

```diff
@@ -128,6 +128,22 @@ async function register ({
 
       return res.json(result)
     })
+
+    router.post('/send-notification', async (req, res) => {
+      peertubeHelpers.socket.sendNotification(1, {
```

Comment:
Most of the type `root` is id `1`, but it may change. It would be better for the tests to send the user id in body